unittests/auth_tests.cpp

```diff
@@ -341,7 +341,7 @@ BOOST_AUTO_TEST_CASE( any_auth ) { try {
 
    const auto spending_priv_key = chain.get_private_key(name("alice"), "spending");
    const auto spending_pub_key = spending_priv_key.get_public_key();
-   const auto bob_spending_priv_key = chain.get_private_key(name("bob"), "spending");
+   //const auto bob_spending_priv_key = chain.get_private_key(name("bob"), "spending");
```

Comment:
Why just comment out, not remove?

Reply:
I'm NOT a fan of commented out code. But the symmetry with the others made me comment it out instead of remove. I'll go ahead and just remove it.